CHANGELOG.md

```diff
@@ -81,7 +81,7 @@ if (accessTokenPayload.jwt === undefined) {
 
 ```tsx
 let jwt = null;
```

Comment:
access token paylaod in session information vs the return from the getAccessTokenPayload is inconsistent.